src/main/scala/wdl4s/Call.scala

```diff
@@ -180,7 +180,7 @@ sealed abstract class Call(val alias: Option[String],
         parentLookup <- Try(parent.lookupFunction(inputs, wdlFunctions, outputResolver, shards, relativeTo)(name))
       } yield parentLookup
 
-      val resolutions = Seq(inputMappingsLookup, declarationLookup, declarationExprLookup, taskParentResolution)
+      val resolutions = Seq(inputMappingsLookup, declarationExprLookup, declarationLookup, taskParentResolution)
```

Comment:
streams are one of a handful of things that I'm starting to think we'll be seeing a lot more of in wdl4s. another example would be vectors instead of the generic seqs due to all of the `++`s and such.